### DISCUSSION TITLE
What’s with the failing tests?

### DISCUSSION THREAD

**Discussion by ThatXliner** [pulsar-edit/.github#45] [Pulsar Support] [answered]
Are they unrelated or what? And, unless I’m mistaken, I see that you guys have migrated Atom to GitHub Actions

**Comment by kaosine** [comment, marked as answer]
They're a work in progress. It's taking awhile to resolve all the issues after updates and removing the old script folder.

**Reply by mauricioszabo** [reply]
Just to add more context, we decided to disable the failing tests for now, because some of these are basically flaky tests that if you run then independently, they pass, and if you run then together with other tests, they fail.

So we'll be probably fixing/rewriting them somehow, but basically it means that while these tests are important, knowing that new code didn't break anything new is more important considering we want to modernize Electron and such :+1:
